pyspark/bigdl/contrib/onnx2bigdl/import_graph.py

```diff
@@ -0,0 +1,130 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+from bigdl.nn.onnx.layer import *
+from bigdl.nn.layer import Identity, Model
+
+from .ops_mapping import _convert_map as convert_map
+
+
+class BigDLGraph(object):
```

Comment:
I think it's a equivalent ONNX graph right ?  Not BigDL graph, loadONNX should eventually return BigDL graph. We should not define a ONNX equivalent class in BigDL, for those states that are different from ONNX, we should hide them in loading and just return BigDL model

Reply:
Hi Jerry, thanks for your time!
Those classes are hidden from users, I have updated the PR with some example codes.
If you still think those classes unnecessary, they could be deleted.

Reply:
Not necessarily delete it if required,  it makes more sense to update the name such as `ONNXGraph` instead of 'BigDLGraph' to avoid confusion.

Reply:
Thanks, changed.